executor/executor_test.go

```diff
@@ -4066,6 +4066,10 @@ func (s *testSuite) TestSplitRegion(c *C) {
 	_, err = tk.Exec(`split table t between (0) and (100) regions 10`)
 	c.Assert(err, NotNil)
 	c.Assert(err.Error(), Equals, "Split table `t` region step value should more than 1000, step 10 is invalid")
+
+	// Test split region by syntax
+	tk.MustExec(`split table t index idx1 by (0),(1000),(1000000)`)
```

Comment:
I think this case has existed in line 3995.

Reply:
done.